benchmark/scenarios/test/test_write_request_load.py

```diff
@@ -167,6 +168,39 @@ def assert_created(returned_datasets):
         d2.addCallback(assert_created)
         s.stop()
 
+    @capture_logging(None)
+    def test_cleanup_deletes_dataset(self, _logger):
+        """
+        ``write_request_load_scenario`` deletes the dataset created by
+        the setup when the scenario is stopped.
+        """
```

Comment:
Some more descriptive names would be good here - e.g. clock rather than c, scenario rather than s.